locale/enUS.lua

```diff
@@ -113,7 +115,10 @@ L['Opens this tutorial again.\nHope you liked it\n\n:)\n\n'] = true;
 L['Buy Items'] = true;
 L['AuctionFaster - Buy'] = true;
 L['Search'] = true;
-L['Filters'] = true;
```

Comment:
Why this one was removed? Was it duplicate?

Reply:
L['Filters'] repeat at line 70